app/src/main/java/com/cornellappdev/android/eatery/model/EateryBaseModel.java

```diff
@@ -144,6 +155,32 @@ public void parseEatery(Context context, boolean hardcoded, AllEateriesQuery.Eat
     mArea = CampusArea.fromShortDescription(eatery.campusArea().descriptionShort());
   }
 
+  public void parseCtEatery(Context context, AllCtEateriesQuery.CollegetownEatery ctEatery) {
+    mName = ctEatery.name();
+    mNickName = ctEatery.name();
+    mBuildingLocation = ctEatery.address();
+    mLatitude = ctEatery.coordinates().latitude();
+    mLongitude = ctEatery.coordinates().longitude();
+    mId = ctEatery.id();
+    mPhoneNumber = ctEatery.phone();
+    mEateryType = ctEatery.eateryType();
+
+    List<PaymentMethod> paymentMethods = new ArrayList<>();
+    if (ctEatery.paymentMethods().brbs())
+      paymentMethods.add(PaymentMethod.fromShortDescription("debit"));
+    if (ctEatery.paymentMethods().cash())
+      paymentMethods.add(PaymentMethod.fromShortDescription("cash"));
+    if (ctEatery.paymentMethods().cornellCard())
```

Comment:
you sure we need this?

Reply:
the fields are still there for ct eateries as well. I can get rid of it but why not keep it